### PR TITLE
Feature/add ingestr

### PR DESCRIPTION
**Pull request by albertobruin**

### CODE REVIEW

File: pkg/ingestr/operator.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+func (o BasicOperator) Run(ctx context.Context, ti scheduler.TaskInstance) error {
+	return o.RunTask(ctx, ti.GetPipeline(), ti.GetAsset())
+}
+
+func (o BasicOperator) RunTask(ctx context.Context, p *pipeline.Pipeline, t *pipeline.Asset) error {
+	return errors.New("Ingestr tasks not implemented")
+}
+
+type IngestrCheckOperator struct {
+	configs *map[pipeline.AssetType]executor.Config
```
Comment (karakanb) [inline pkg/ingestr/operator.go:27]: why is the value here of type `executor.Config` but not an actual column check operator instance?

File: pkg/ingestr/operator.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+func (i IngestrCheckOperator) Run(ctx context.Context, ti scheduler.TaskInstance) error {
+	asset := ti.GetAsset()
+	assetType, err := asset.GetIngestrDestinationType()
```
Comment (karakanb) [inline pkg/ingestr/operator.go:32]: I think this is very specific to the operator itself, I wouldn't put this method on the asset itself

File: pkg/ingestr/operator.go (hunk 1 above)
Comment (albertobruin) [reply pkg/ingestr/operator.go:27]: The idea is that the ingestr check operator needs to be able to perform checks for different types of destination. To illustrate with an example, imagine you have a pipeline where you have 2 ingestr assets, one has a postgres destination and another one has a mssql destination. The are both ingestr so they would use this check operator but the checks need to be different

File: pkg/ingestr/operator.go (hunk 2 above)
Comment (albertobruin) [reply pkg/ingestr/operator.go:32]: Thing is that it's needed not only in the operator but also outside so I thought it was a nice abstraction. Will create a function function in the `ingestr` package instead like

```go
func GetIngestrDestinationType(asset Asset) (AssetType, error)
```

**Pull request merged**